packages/docs-site/src/content/docs/docs/plugins/1password.mdx

```diff
@@ -80,39 +85,70 @@ Consider how you want to organize your vaults and service accounts. You might ha
 
 ## Add your items
 
-DMNO supports a few different ways to reference items in 1Password. You can use a direct link to the item, the item's UUID, or a reference to the item in the format `op://vaultname/itemname/path`. We also support, storing the items in one large blob, similar to your existing `.env` file(s). This allows you to easily transition from a `.env` file to 1Password as a secure vault.
+DMNO supports a few different ways to reference items in 1Password.
 
-### Using a env blob
+### Using a env blob (recommended)
 
-If you have a single item that contains all your secrets, you can reference that item in your config. In the example below, we're referencing an item called `env` that contains all our secrets and matching based on the key, in this case `ITEM_PULLING_FROM_ENV_BLOB_ITEM`.
+Managing lots of individual 1password items and connecting them to your config can be a bit tedious, so we recommend storing multiple items together in a `.env` style text blob. Using this method, we'll have a single 1password "item" that can have one text entry per "service" containing the `.env` blob. This would be similar to applying a `.env.local` file as overrides, except they are secured and shared via 1password. This also makes it incredibly easy to migrate from using local `.env` files.
```

Comment:
```suggestion
Managing lots of individual 1password items and connecting them to your config can be a bit tedious, so we recommend storing multiple items together in a `.env` style text blob. Using this method, we'll have a single 1password item that can have one text entry per service containing the `.env` blob. This would be similar to applying a `.env.local` file as overrides, except they are secured and shared via 1password. This also makes it incredibly easy to migrate from using local `.env` files.
```